### DISCUSSION TITLE
LORA saving previews messed up

### DISCUSSION THREAD

**Discussion by andykaufseo** [d8ahazard/sd_dreambooth_extension#669] [Q&A]
With regular training saving previews works just fine, according to the settings entered in the fields.

But with LORA checked things are messed up, here's just an example : 

400 epochs / save training steps every 20 epochs

First preview got saved at epoch 210 and then it constantly saves every 20 epochs as it should. Not sure why it starts generating previews so late tho. 

This started happening 2-3 commits ago (i guess when LORA got implemented)

**Comment by andykaufseo** [comment]
Been doing a few more tests with LORA, one ran for 5k steps (100 epochs) and i got no sample generated. And one that ran for 500 epochs (but fewer images) and i got only 3 samples generated. All while set to generate samples every 20 epochs

And the same thing for non lora training - after 100 epochs no sample generated (set to generate samples every 20 epochs) - batch size 4, apply horizontal flip checked (i see these somehow seem to influence the generation of samples, but i can't find any consistent data about it)

**Comment by RevyaM** [comment]
This problem has been here for a while, I believe. 
The only workaround is to save previews via button in the UI.
